lib/mpl_toolkits/axisartist/floating_axes.py

```diff
@@ -245,30 +235,21 @@ def __init__(self, aux_trans, extremes,
                          tick_formatter1=tick_formatter1,
                          tick_formatter2=tick_formatter2)
 
-
```

Comment:
should these go away as well?

Reply:
there's an update_grid_finder in grid_helper_curvelinear so I'll leave this one in